### PR TITLE
Blog: why and who post

### PR DESCRIPTION
**Pull request by mfridman**
Thoughts, comments, feedback welcome.

If you have any other questions you came across let know.

### CODE REVIEW

**Review by Gonzih — APPROVED** [review]
Well written blog post!

File: content/blog/gocon-who-and-why.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  <div class="column">
+    <a href="https://crowdriff.com/"><img src="/img/sponsors/crowdriff-logo.png" alt="sponsor crowdriff" width="240" height="auto"></a>
+  </div>
+  <div class="column">
+    <a href="https://kirasystems.com/"><img src="/img/sponsors/kira-logo.svg" alt="sponsor kirasystems" width="180" height="auto"></a>
+  </div>
+  <div class="column">
+    <a href="https://www.manifold.co/gocon?utm_campaign=gocon&utm_source=gocon&utm_medium=sponsorship"><img src="/img/sponsors/manifold-logo.png" alt="sponsor manifold" width="240" height="auto"></a>
+  </div>
+</div>
```
Comment (arbourd) [inline content/blog/gocon-who-and-why.md:70]: Maybe we should redirect to the sponsors chunk instead of re-listing them here?

**Review by arbourd — APPROVED** [review]
Amazing work @mfridman.

What about also adding something like

> What can I do to get involved with GoCon?

We've had a lot of people ask us this and it might be worth mentioning how excited folks can help out!

Some ideas:
- Get colleagues and friends involved
- Submit to the CFP, everyone (even beginners) are encouraged to participate
- Come to the event? haha

**Comment by mfridman** [comment]
Good points, I'll add that question .. indeed it has come up a few times.

**Pull request merged**